minio/policy.py

```diff
@@ -119,6 +131,10 @@ def _new_statements(policy, bucket_name, prefix=''):
             _new_object_statement(policy, bucket_name, prefix))
 
 
+def resource_startswith(prefix, resources):
+    return list(filter(lambda r: r.startswith(prefix), resources))
```

Comment:
```
In [51]: bool(filter(lambda r: r.startswith('c'), ['aa', 'bb']))
Out[51]: True
```
In python 3, filter returns filter object and its bool is always true. So we have to convert to list.

Documented `_filter_resources`. Thanks.